ReactiveCocoa/Swift/Property.swift

```diff
@@ -95,6 +95,57 @@ extension MutableProperty: SinkType {
 	}
 }
 
+/// Wraps a `dynamic` property, or one defined in Objective-C, using Key-Value
+/// Coding and Key-Value Observing.
+///
+/// Use this class only as a last resort! `MutableProperty` is generally better
+/// unless KVC/KVO is required by the API you're using (for example,
+/// `NSOperation`).
+@objc public final class DynamicProperty: NSObject, PropertyType {
+	public typealias Value = AnyObject?
```

Comment:
That would prevent it from being exposed to Objective-C (which I thought might be handy).

And, ultimately, we're going to have to cast somewhere, because KVC and KVO just use `AnyObject?` for everything.